### PR TITLE
Users/stevenh/add manifest

### PR DESCRIPTION
**Pull request by stevenhorsman**
Part of zowe/zowe-install-packaging/issues/1692

- Add manifest to explorer-jes component
- refactored to move app as sibling of bin directory
- updated script to reflect explorer-ui-server changes
- some minor bug fixes

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+# ===== Zowe Component Manifest Definition =====
+# Component name
+# this will be the folder name under <zowe-runtime>/components folder
+# please note this value is not same as the service id we defined in apiml-static-registration.yaml.template
```
Comment (jackjia-ibm) [inline manifest.yaml:5]: We can remove this line.

File: apiml-static-registration.yaml.template [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - serviceId: explorer-jes
+    title: JES Explorer
+    description: IBM z/OS Jobs UI service
+    catalogUiTileId:
+    instanceBaseUrls:
+      - https://${ZOWE_EXPLORER_HOST}:${JES_EXPLORER_UI_PORT}/
+    homePageRelativeUrl:
+    routedServices:
+      - gatewayUrl: ui/v1
+        serviceRelativeUrl: ${EXPLORER_PLUGIN_BASEURI}
```
Comment (jackjia-ibm) [inline apiml-static-registration.yaml.template:11]: We need to make sure this variable has a value with current instance.env when we want to parse the template with new component launcher script. Or pick value from `configs.baseUri`.

**Review by jackjia-ibm — APPROVED** [review]

File: manifest.yaml (hunk 1 above)
Comment (stevenhorsman) [reply manifest.yaml:5]: Deleted the lines

File: apiml-static-registration.yaml.template (hunk 2 above)
Comment (stevenhorsman) [reply apiml-static-registration.yaml.template:11]: Yep - I've updated it and added a note to indicate that when we start using template injection we need to get this value from the parsed component config or directly from manifest.yaml

**Review by jackjia-ibm — APPROVED** [review]
Looks all good! Thanks a lot

**Pull request merged**